web/src/lib/InstallerClient.js

```diff
@@ -96,10 +114,22 @@ export default class InstallerClient {
     )
   }
 
+  async getOption(name) {
+    try {
+      const [{ v: option }] = await this.client().call(
```

Comment:
I love it :stuck_out_tongue_winking_eye: But lets drop a useful link here

* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#nested_object_and_array_destructuring